config.yml

```diff
@@ -84,6 +84,8 @@ default:
             icgc_dart: "all_the_things/cnv_master-1.0/merges/{seq_type}--{projection}.seg"
             template: "all_the_things/cnv_master-1.0/merges/{seq_type}--{projection}.seg"
         sv: "icgc_dart/cbioportal-1.0/01-filtered-manta/genome-grch37/all_merged_sv_info.tsv"
+        sv_manta:
```

Comment:
I thought this was already in the config. Maybe just in my branch though?

Reply:
Yeah, I did not have it in my branch, that is for sure.